CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the docstring information for the `FBetaMultiLabelMeasure` metric.
 - Various fixes for Python 3.9
 - Fixed the name that the `push-to-hf` command uses to store weights.
+- Support for inferior operating systems when making hardlinks
```

Comment:
Shots fired 🤣

Reply:
[Windows does actually support hardlinks](https://docs.microsoft.com/en-us/windows/win32/fileio/hard-links-and-junctions).